### ISSUE TITLE
Freeze bug

### ISSUE THREAD

**Issue by ubidval** [Aymkdn/html-to-pdfmake#197]
Hi,
I have a freeze bug when the HTML is converted to a pdfmake definition. When we have a row with several lines (greater than 2), JS freezes.

The HTML code used :
```html
<table style="width: 100%;" data-pdfmake='{ "dontBreakRows": true }'>
    <tr>
        <th colspan="10" rowspan="1">Pourcentage du capital détenu</th>
        <th colspan="12" rowspan="1">Type de titre</th>
    </tr>

    <tr>
        <td colspan="10" rowspan="3">99.00</td>
        <td colspan="12" rowspan="1">Action simple</td>
    </tr>

    <tr>
        <td colspan="12" rowspan="1">Action de préférence</td>
    </tr>

    <tr>
        <td colspan="12" rowspan="1">Total</td>
    </tr>
</table>
```

**Comment by Aymkdn** [comment]
I've just left, for one week of vacation. If I can find a computer I'll see what I can do. So, please be patient.

**Comment by Aymkdn** [comment]
That should be fixed with v2.5.1

**Comment by ubidval** [comment]
Thanks
